suse_caasp

```diff
@@ -382,3 +407,38 @@ if check_rpm salt-minion && [ -e /usr/bin/salt-minion ]; then
     find_and_log_files $OF /var/log/salt
 fi
 plugin_message Done
+
+# checks that only apply to the admin node
+if on_admin && check_rpm docker && [ -e /usr/bin/docker ]; then
```

Comment:
The checks should be done in a different order: `on_admin` is already making use of the `docker` binary, hence the check for existence of it should be done immediately.